libretroshare/src/retroshare/rsinit.h

```diff
@@ -276,6 +276,10 @@ class RsAccounts
 	 * @return true on success, false otherwise
 	 */
 	static int GetPGPLogins(std::list<RsPgpId> &pgpIds);
+     /**
+	  * @jsonapi{development}
```

Comment:
you need to add an entry for each parameter. See the other methods in rsinit.h

Reply:
```/** @brief Lock profile directory
	 * param[in] accountDir account directory to lock
	 * param[out] lockFilePath path of the created lock-file
	 */
```
You mean to say entry like this i need to add

Reply:
yes